client/my-sites/preview/controller.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WebPreviewContent from 'components/web-preview/content';
+
+export default {
+	preview: function( context, next ) {
+		context.primary = (
+			<div style={ { height: '100%' } }>
+				<WebPreviewContent
+					previewUrl={ `https://${ context.params.site }/?iframe=true&preview=true` }
```

Comment:
`context.params.site` is value from route `/preview/:site`. I think router makes sure it exists but it can, of course, contain invalid data. This is a weak implementation that doesn't take into count many factors and we will replace later once we find a good solution to get https address. For now, I think this should be good enough for our testing.